plugins/accuracy.cc

```diff
@@ -86,6 +86,11 @@ AccuracyConfigDescription()
               "test set is very large and aggregate statistics for each unique score is "
               "sufficient, for instance to generate a ROC curve. This has no effect "
               "for other values of `mode`.", false);
+    addField("accuracyOverN", &AccuracyConfig::accuracyOverN,
```

Comment:
I don't think it's a lot of work, and it avoids us shipping something with all the technical work done but a poor UX.